src/Dialect/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 add_subdirectory(Krnl)
-add_subdirectory(ONNX)
\ No newline at end of file
+add_subdirectory(ONNX)
+if (INCLUDE_ONNX_ML)
```

Comment:
I think we should be building ONNX ML unconditionally. It's one of the dialects we supports. MLIR builds all its dialects and then it's up to the user which ones get registered and used.